README.md

```diff
@@ -15,8 +15,19 @@ Estimating the geographical range of a species from sparse observations is a cha
 # Getting Started
 
 ## :hatched_chick: Installation for local development
-Make sure you update your local branch to the latest.
+1. Clone the repository `git clone git@github.com:UMassCDS/ds4cg2024-inaturalist.git`
 
+2. Navigate to cloned project's root, run `git submodule init` and then `git submodule update`
```

Comment:
please update the docker file, add these commands there before pre-trained models are downloaded there.

Thank you!

Reply:
I am working on adding these requests, it's a little more complicated because git is trying to authenticate while building the image

Reply:
Ok, I managed to get it working, auth was a problem because I had the url that wanted the ssh auth. I will update the branch, if there is still a problem like auth failed, while building docker images, the solution might be in .git/modules/src/backend/sinr/config, in the config file you may need to change the url to http url instead of ssh